### ISSUE TITLE
Edit object as JSON

### ISSUE THREAD

**Issue by mattlag** [mattlag/Glyphr-Studio#166]
Could pretty easily create a pane that shows that objects JSON, then re-import it when the user hits a save button.

**Comment by davelab6** [comment]
Would be great if it was 'live', similar to jsbin.com jsfiddle.net etc

And such a data-structure view complements a 'scripting' panel that executes JS to modify the JSON, as in https://github.com/mattlag/Glyphr-Studio/issues/157 :)

**Comment by mattlag** [comment]
Yep - agreed.  Live updating as the user types may be overkill, but it may be okay.  For all of the edits, i'd basically be creating a new object and displaying it in place of the old one.  There might be some invalid JSON as they type that would just cause the object to revert to the default object.

**Comment by davelab6** [comment]
I know you've made it so far with minimal use of libraries, but
https://github.com/jsbin/jsbin and https://github.com/ether/etherpad-lite
are open books if you don't want to plumb them in :)

**Comment by mattlag** [comment]
Cool, thanks for the links, i'll check them out.  I am much more likely to use libraries if they aren't core to what i'm interested in :-D
